docs/content/preview/yugabyte-cloud/cloud-clusters/cloud-maintenance.md

```diff
@@ -62,14 +62,20 @@ To set the maintenance exclusion period for a cluster:
 
 ## What to expect during maintenance
 
-Yugabyte performs rolling maintenance and upgrades on [fault tolerant](../../cloud-basics/create-clusters-overview/#fault-tolerance) clusters with zero downtime. However, the cluster is still subject to the following:
+The impact of maintenance on a cluster depends on its topology and [fault tolerance](../../cloud-basics/create-clusters-overview/#fault-tolerance).
 
-- Dropped connections - Connections to the stopped node are dropped. Verify your connection pool, driver, and application to ensure they handle dropped connections correctly. Any failures need to be retried.
+### Fault tolerance none
+
+For any cluster with fault tolerance of none (for example, single node and sandbox clusters), the cluster will be briefly unavailable while the node is patched and then restarted.
```

Comment:
It may be worth adding the "why" here. Maybe we just call out that its due to a replication factor of 1 and that this ties directly into the fault tolerance of the cluster, then link here to learn more - https://docs.yugabyte.com/preview/architecture/docdb-replication/replication/#replication-factor